src/main/bluesky/creator.py

```diff
@@ -2,9 +2,9 @@
 
 
 # temp file setup
-inputfile = "Data/Flights3.csv"
+inputfile = "Data/Flights4.csv"
```

Comment:
Do you want to keep the temp file setup in main?

Reply:
no i fixed that already that was temp